docs/user/app_feature_intended.md

```diff
@@ -29,26 +29,23 @@ or
 
 ## Adding Jinja2 Filters to the Environment.
 
-This plugin follows [Nautobot](https://nautobot.readthedocs.io/en/stable/plugins/development/#including-jinja2-filters)
-in relying on [django_jinja](https://niwinz.github.io/django-jinja/latest/) for customizing the Jinja2 Environment.
-Currently, only filters in the django_jinja Environment are passed along to
-the Jinja2 Template Environment used by Nornir to render the config template.
+This plugin follows [Nautobot](https://nautobot.readthedocs.io/en/stable/plugins/development/#including-jinja2-filters) in relying on [django_jinja](https://niwinz.github.io/django-jinja/latest/) for customizing the Jinja2 Environment. Currently, only filters in the `django_jinja` Environment are passed along to the Jinja2 Template Environment used by Nornir to render the config template.
 
 ### Adding Filters In Nautobot Config
 
-Nautobot documents using the `@django_jinja.library.filter` decorator to register functions as filters with django_jinja.
-However, users of plugins are not able to define plugins in the specified jinja2 filter file that is loaded into the Jinja2 Environment.
-There are several alternative ways to have functions registered as filters in the django_jinja environment;
-below demonstrates defining decorated functions in a separate file, and then importing them in the `nautobot_config.py` file.
-This method requires that the file is in a path that is available to Nautobot's python environment.
+Nautobot documents using the `@django_jinja.library.filter` decorator to register functions as filters with `django_jinja`. However, users of plugins are not able to define plugins in the specified jinja2 filter file that is loaded into the Jinja2 Environment.
+
+There are several alternative ways to have functions registered as filters in the `django_jinja` environment; below demonstrates defining decorated functions in a separate file, and then importing them in the `nautobot_config.py` file. This method requires that the file is in a path that is available to Nautobot's python environment.
 
-> django_jinja documents adding filters in the `TEMPLATES` config section;
-> since Nautobot sets the `TEMPLATES` config section and does not document this in optional settings,
-> it is recommended to only use the `@django_jinja.library.filter` decorator.
+!!! note
+    `django_jinja` documents adding filters in the `TEMPLATES` config section;
```

Comment:
Why keep these lines separate, and combine all of the other lines that are separate?

Reply:
I missed this one, no other reason :)